### PR TITLE
BI-9543 Add liquidators details types and mappings

### PR DESCRIPTION
**Pull request by phudson-ch**
- Add liquidators details types and mappings
- Fix Typescript warnings
- Remove unused imports and types

Resolves: [Add liquidators_details field: api-sdk-node](https://companieshouse.atlassian.net/browse/BI-9543)

### CODE REVIEW

File: src/services/order/order/mapping.ts [hunk 1]
```diff
@@ -134,7 +137,8 @@ export default class OrderMapping {
                 secretaryDetails: secretaryDetails,
                 directorDetails: directorDetails,
                 forename: itemResource.forename,
-                surname: itemResource.surname
+                surname: itemResource.surname,
+                liquidatorsDetails
```
Comment (kinpang-CH) [inline src/services/order/order/mapping.ts:141]: Make sure undefined values are filtered from the object being returned

File: test/services/certificates/service.spec.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import chaiHttp from "chai-http";
 import CertificateService from "../../../src/services/order/certificates/service";
 import { RequestClient, HttpResponse } from "../../../src/http";
 import { CertificateItemResource, CertificateItemPostRequest, CertificateItemPatchRequest } from "../../../src/services/order/certificates/types";
+import exp = require("constants");
```
Comment (kinpang-CH) [inline test/services/certificates/service.spec.ts:9]: Not being used

**Review by kinpang-CH — COMMENTED** [review]
Address PR comments

File: test/services/certificates/service.spec.ts (hunk 2 above)
Comment (phudson-ch) [reply test/services/certificates/service.spec.ts]: Removed as advised.

**Review by cgregoryCH — APPROVED** [review]

**Review by dlgroves-ch — APPROVED** [review]

**Review by kinpang-CH — APPROVED** [review]

**Pull request merged**